cves/CVE-2017-5664.yml

```diff
@@ -43,7 +43,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The Error page mechanism did not go to the custom error page which was
+  configured for a specific error. It instead leads to unexpected static Error pages.
```

Comment:
Instead of a static error page, do you mean do a generic error page that is generated by the browser perhaps?